pkg/util/env/env.go

```diff
@@ -53,6 +53,16 @@ func GetSMCPVersion() version.Version {
 	return version.ParseVersion(getenv("SMCP_VERSION", "v2.5"))
 }
 
+func GetOperatorVersion() version.Version {
+	return version.ParseVersion(getenv("OPERATOR_VERSION", "v2.5"))
+}
```

Comment:
Is this used anywhere? If not, it's better to remove it.

Reply:
It's used now :)